packages/iris-grid/src/IrisGridModel.ts

```diff
@@ -291,6 +291,14 @@ abstract class IrisGridModel<
     return false;
   }
 
+  /**
+   * @param index The column index to check
+   * @returns Whether the column is sortable
+   */
+  isColumnSortable(index: ModelIndex): boolean {
```

Comment:
You don't need these methods on `IrisGridModel`, as you can just access `isSortable` on the `Column` itself.

I guess it is strange that there is an `isColumnFrozen` method, rather than accessing that on the `Column` itself. I would prefer to make that one the odd one out though.

Reply:
The IrisGridProxyModel complains if the isColumSortable method is erased from here

Reply:
Right, you'd need to remove it from `IrisGridProxyModel` as well.
Making this property optional is annoying though, as it does require adding the `?? true` everywhere that accesses it... so perhaps leave this method in, and make the other changes to default to `?? true` that I mentioned (instead of `?? false`) and opt to use `model.isColumnSortable(column)` instead of the property on the column directly when possible.